docs/build.md

```diff
@@ -0,0 +1,128 @@
+# Build instructions
+
+The pipeline requires Python version 3.5.
+
+## Python 3.5 installation (optional)
+The instructions below will be useful:
+* In case you have a newer version and need to run the pipeline you can follow the next steps to install Python 3.5
+without replacing you default `python`;
+* If you are running the pipeline on the `ebi-cli` cluster, which currently only supports Python 3.4.
+
+Adjust `VERSION` and `INST_PATH` if needed.
+
+```bash
+VERSION=3.5.6
+INST_PATH=${HOME}/software/python-${VERSION}
+mkdir -p ${INST_PATH}
+cd ${INST_PATH}
+wget https://www.python.org/ftp/python/${VERSION}/Python-${VERSION}.tgz
+tar zxfv Python-${VERSION}.tgz
+mv Python-${VERSION}/* .
+rmdir Python-${VERSION}
+```
+
+### Root installation
+
+This way is easier as it doesn't require you to set up any paths manually. Package installation is also less likely
+to go wrong if you're using this way.
+
+You need to have zlib development headers installed. Command for Debian/Ubuntu is `sudo apt-get -y install
+zlib1g-dev`.
+
+```bash
+./configure --with-zlib=/usr/include --enable-optimizations
+make -j `nproc` altinstall
+```
+
+You can now invoke the new installation as `python3.5`.
+
+### Non-root installation
+
+This is the only way which will work on the `ebi-cli` cluster. It is also useful if you need to have more than one
+minor version installed, e. g. Python 3.5.0 and 3.5.6, simultaneously.
+
+You can optionally add `--enable-optimizations` flag to `./configure` call (as in the root installation way), but
+note that it will significantly slow down the build because the cluster is using NFS, and the build may take up to a
+few hours to complete.
+
+```bash
+./configure --prefix=$INST_PATH
+make -j `nproc`
+make -j `nproc` install
+```
+
+You can invoke this installation by using a direct path, `${INST_PATH}/python`. In order to temporarily set it as
+default (and invoke simply as `python`), configure the paths:
+
+```bash
+export PATH=$INST_PATH:$PATH
+export PYTHONPATH=$INST_PATH
+```
+
+To install packages for this installation you won't be able to use `pip` as a standalone command and instead you'll
```

Comment:
```suggestion
You won't be able to use `pip` as a standalone command to install packages in this mode. Instead you'll
```

Reply:
Your version is much more readable, used it